incubator/redis-cache/templates/ss.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: "{{ template "fullname" . }}"
+  labels:
+     app: {{ template "fullname" . }}
```

Comment:
Okay will update but what would have been even more helpful is if i got this comment few weeks ago, when i originally updated this following your review comment.